api/api-npm/package.json

```diff
@@ -25,8 +27,21 @@
     "url": "https://github.com/fluencelabs/aqua/issues"
   },
   "homepage": "https://github.com/fluencelabs/aqua#readme",
+  "scripts": {
+    "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --detectOpenHandles"
+  },
+  "optionalDependencies": {
```

Comment:
After reading this https://docs.npmjs.com/cli/v7/configuring-npm/package-json#optionaldependencies
I am not sure these should optional, cause you dont handle the case the dependencies are missing. Also the are actually required if you want to use gather functions you created, they will not work without these dependencies, right?